src/Adapter/Module/ModuleDataProvider.php

```diff
@@ -114,6 +126,7 @@ public function isModuleMainClassValid($name)
         $content = preg_replace('/\n[\s\t]*?use\s.*?;/', '', $file);
         $content = preg_replace('/\n[\s\t]*?namespace\s.*?;/', '', $content);
```

Comment:
A issue has been created for this point: http://forge.prestashop.com/browse/BOOM-606